### PR TITLE
Prod Smoke Test Query

### PR DESCRIPTION
**Pull request by amarjandu**
Remove possibility for tombstoned bundles to be in the prod-smoketest

### CODE REVIEW

File: tests/test_prod_smoketest.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class ProdSmoketest(BaseSmokeTest):
     def setUpClass(cls):
         super().setUpClass()
         cls.url = "https://www.example.com"
-        cls.query = {"query": {"bool": {"must": [{"term": {"admin_deleted": "true"}}]}}}
+        cls.query = {"query": {"bool": {"must_not": [{"term": {"admin_deleted": "true"}}]}}}
```
Comment (xbrianh) [inline tests/test_prod_smoketest.py]: Won't this change interfere with the subscription notification testing?

We may need to define two queries.

Comment (amarjandu) [reply tests/test_prod_smoketest.py]: Due to the test not creating/destroying data, it only checks if the subscription can successfully be created/retrieved. From my tests locally it works successfully with the new query. I can add in the older one if requested.  

Screen Shot 2019-04-30 at 12 46 36 PM

Comment (xbrianh) [reply tests/test_prod_smoketest.py]: OK, I understand.

Please make it clear (via comment or naming scheme) that the subscription query is not expected to be used.

Comment (amarjandu) [reply tests/test_prod_smoketest.py]: added comment

**Pull request merged**